docs/guides/hooks-list.rst

```diff
@@ -585,6 +585,14 @@ Other
 	"container_guid" (int) are provided. An empty entity value generally means the form is to
 	create a new object.
 
+**elgg_channels:send, <channel_name>**
```

Comment:
A WS plugin would register for "all" on both hooks I would think. It wouldn't need to manage queues in the DB or anything, just pass the message to the long-running push server and forget about it.